cypress/integration/home-spec.js

```diff
@@ -0,0 +1,130 @@
+import movies from '../fixtures/movies.json'
+
+describe('Rancid Tomatillos home page data load flow', () => {
+    
+    beforeEach(() => {
+        cy.fixture('movies.json').as('movies')
+    })
+    
+    it('When the user visits the home page, the title of the site should be displayed in the header.', () => {
+        cy.visit('localhost:3000')
+        cy.get('.navbar').contains('Rancid Tomatillos')
+    })
+    
+    it('When a user visits the page, the movie posters should be display in a grid.', () => {
+        cy.intercept('GET',  'https://rancid-tomatillos.herokuapp.com/api/v2/movies', {
+            statusCode: 200,
+            ok: true,
+            body: { movies },
+        })
+        cy.visit('localhost:3000')
+        cy.get('.grid-posters')
+            .should('have.attr', 'src')
+            .and('equal', 'https://image.tmdb.org/t/p/original//c59eplVELdwrUfGBUAZVin3HfaL.jpg')
```

Comment:
This was how I went about testing during the last inning, but I attempted to make the URL dynamic this time around.  Much bigger pain, as it required using `cy.fixture()` much more often than I would've preferred.  Would definitely like to revisit my tests to refactor!